components/gateway/go.mod

```diff
@@ -33,3 +33,5 @@ require (
 	gopkg.in/check.v1 v1.0.0-20200902074654-038fdea0a05b // indirect
 	gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b // indirect
 )
+
+require github.com/avast/retry-go v3.0.0+incompatible // indirect
```

Comment:
Why is this outside of the `require` block above?

Reply:
Automation :-(